specifications/sep-payment-channel-transactions.md

```diff
@@ -229,45 +227,61 @@ payments to one another within the channel.
 For example, if the channel initial state is $30 of which $10 belongs to I and
 $20 belongs to R, the first closing transaction will disburse $10 to I and $20
 to R. If I makes a payment of $2 to R, this update process will involve I and R
-agreeing on a new declaration and aclosing transaction that supersedes all
-previous declaration and closing transaction and that will disburse $8 to I and
-$22 to R.
+agreeing on new declaration, payment, and closing transactions that supersede
+all previous declaration, payment, and closing transaction and that will
+disburse $8 to I and $22 to R.
 
 To update the payment channel state, the participants:
 
 1. Increment i.
 2. Sign and exchange a closing transaction C_i.
-3. Sign and exchange a declaration transaction D_i.
+3. Sign and exchange zero or more payment transactions P_{i,a}.
+4. Sign and exchange a declaration transaction D_i.
 
-It is important that D_i is signed after C_i because D_i will invalidate any
-previously signed C_i. If I and R signed and exchanged D_i first either party
-could prevent the channel from closing without coordination by submitting D_i
-and refusing to sign C_i.  The participants would not be able to close the
-channel, or regain control of the accounts, and the assets within without
-coordinating with each other.
+It is important that D_i is signed after C_i and P_i because D_i will invalidate
```

Comment:
I think this is a good change, but I think we should add some specificity to what's being exchanged. They shouldn't exchange signatures, but they may still exchange transactions.

```suggestion
It is critical that signatures for D_i are exchanged after C_i and P_i because D_i will invalidate
```

Wdyt?

Reply:
I took this change to a new PR since it is independent to the change this PR is introducing. Could you review https://github.com/stellar/experimental-payment-channels/pull/123?

Reply:
Sure, thanks!